packages/ethereum-contracts/tasks/bundled-abi-contracts-list.json

```diff
@@ -13,8 +13,8 @@
     "IFlowNFTBase", "FlowNFTBase",
     "IConstantInflowNFT", "ConstantInflowNFT",
     "IConstantOutflowNFT", "ConstantOutflowNFT",
-    "IPoolAdminNFT",
-    "IPoolMemberNFT",
+    "IPoolAdminNFT", "PoolAdminNFT",
+    "IPoolMemberNFT", "PoolMemberNFT",
```

Comment:
please also add `IGeneralDistributionAgreementV1` and `GeneralDistributionAgreementV1`

Reply:
done